pom.xml

```diff
@@ -31,13 +31,72 @@
   </scm>
 
   <properties>
-    <jenkins.version>2.222.1</jenkins.version>
-    <java.level>8</java.level>
+    <!-- Dependency org.jenkins-ci.plugins:mailer:jar:412.v7deeda_155287 requires Jenkins 2.331 or higher. -->
```

Comment:
It sounds strange to me that mailer-plugin requires so newer core.

* The latest stable mailer-plugin looks 408.vd726a_1130320 and requires 2.289.1.
    * https://plugins.jenkins.io/mailer/
* What requires mailer-plugin? I believe mailer-plugin isn't used from build-timeout.

Reply:
Recommend target core is described here: https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/